test/hotspot/jtreg/vmTestbase/nsk/jdi/EventSet/resume/resume008.java

```diff
@@ -137,6 +137,17 @@ protected void testRun() {
 
             EventIterator eventIterator = eventSet.eventIterator();
             Event newEvent = eventIterator.nextEvent();
+            display("       got new event: "  + newEvent);
+
+            // Make sure any spurious ThreadStartEvent got filtered out.
+            if (newEvent instanceof ThreadStartEvent) {
+                ThreadStartEvent t = (ThreadStartEvent)newEvent;
+                if (!t.thread().name().equals(resume008a.THREAD_NAME_PREFIX + i)) {
+                    setFailedStatus("ERROR: ThreadStartEvent is not for expected thread:  " +
+                                    t.thread().name());
+                    return;
+                }
+            }
```

Comment:
Not your typo here, but just below on line 157 now there's the "propety" spelling error which was in the failure logs.